rrfs-test/IODA/yaml/prepbufr_adpupa.yaml

```diff
@@ -70,8 +69,6 @@ observations:
               - scale: 100
           pressureTunedError:
             query: "*/PRSLEVEL/P___INFO/P__POSTP/POETU"
-            transforms:
-              - scale: 100
```

Comment:
Should we keep lines 73 and 74 in the updated yaml file?

Reply:
Yes, I must've deleted that by accident. Thanks for catching that. I will fix.